File: angular-client/src/app/setting/txmortality/txmortality.criteria.ts
import { Params } from '@angular/router';

import { AbstractCriteria } from '../../domain/criteria/abstract-criteria';

export class TxMortalityCriteria extends AbstractCriteria {
  name: string;

  populateFromParams(params: Params) {
    this.doPopulateFromParams(params, [], ['pageSize', 'pageNumber'], [], []);
  }

  toParams() {
    return this.doToParams(this);
  }
}

File: couch-server/src/main/java/com/fronde/server/services/txmortality/TxMortalityCriteria.java
package com.fronde.server.services.txmortality;

import com.fronde.server.domain.criteria.AbstractCriteria;

public class TxMortalityCriteria extends AbstractCriteria {

  private String name;

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

}
